### PR TITLE
New EyeDropper feature

### PR DESCRIPTION
**Pull request by captainbrosset**

### CODE REVIEW

File: features/eyedropper.yml [hunk 1]
```diff
@@ -0,0 +1,5 @@
+name: EyeDropper
+description: The EyeDropper API creates an eyedropper tool that allows users to select a color from their screen.
+spec: https://wicg.github.io/eyedropper-api/#eyedropper-interface
+compat_features:
+  - api.EyeDropper
```
Comment (foolip) [inline features/eyedropper.yml:5]: Can you list all 4 BCD keys that are listed by `npm run traverse | grep EyeDropper` in BCD? (Something like this is usually the starting point for me.)

File: features/eyedropper.yml [hunk 2]
```diff
@@ -0,0 +1,5 @@
+name: EyeDropper
+description: The EyeDropper API creates an eyedropper tool that allows users to select a color from their screen.
+spec: https://wicg.github.io/eyedropper-api/#eyedropper-interface
```
Comment (foolip) [inline features/eyedropper.yml:3]:
```suggestion
spec: https://wicg.github.io/eyedropper-api/
```

File: features/eyedropper.yml [hunk 3]
```diff
@@ -0,0 +1,5 @@
+name: EyeDropper
+description: The EyeDropper API creates an eyedropper tool that allows users to select a color from their screen.
```
Comment (foolip) [inline features/eyedropper.yml:2]:
```suggestion
description: The `EyeDropper` API opens an eyedropper tool that allows users to select a color from their screen.
```

**Review by foolip — COMMENTED** [review]
I'm not sure if I should suggest `eye-dropped` instead of `eyedropper`. The spec strangely says `EyeDropper` instead of `Eyedropper`. But https://helpx.adobe.com/photoshop/using/tool-techniques/eyedropper-tool.html suggests that web developers might be more familiar with "Eyedropper" than "Eye dropper".

File: features/eyedropper.yml (hunk 1 above)
Comment (captainbrosset) [reply features/eyedropper.yml:5]: Is `api.EyeDropper.secure_context_required` really needed? I thought this kind of usage requirement was handled at a different level in BCD, not as a compat key.

Comment (foolip) [reply features/eyedropper.yml:5]: Oh, that changed the versions from 95 to 96. That doesn't seem right to me, the feature not working over HTTP isn't essential in any way. So let's exclude the key, after all.

**Review by ddbeck — APPROVED** [review]
Thank you @captainbrosset! The test failure here is due to https://github.com/web-platform-dx/web-features/pull/1182, so this can now merge failing tests notwithstanding. Thanks again!

**Pull request merged**